generate.py

```diff
@@ -79,6 +82,30 @@ def render_item(self, slug, entry):
         with open(os.path.join(self.output_dir, slug + '.html'), 'w') as f:
             f.write(template.render(title=title, entry=entry, authors=authors, **self.nerd_dict))
 
+    def render_sitemap(self):
+        datestr = time.strftime('%Y-%m-%d', time.gmtime())
+
+        pages = []
+        pages.append({'url': self.domain, 'lastmod': datestr})
+        pages.extend([
+            {
+                'url': self.domain + '/{}.html'.format(page),
+                'lastmod': datestr
+            }
+            for page in ('conferences', 'trainings', 'talks')
+        ])
+        pages.extend([
+            {
+                'url': self.domain + '/{}.html'.format(slug),
+                'lastmod': datestr
+            }
+            for slug in self.items
+        ])
```

Comment:
I smell duplication.

for page in ( ... ) + self.items?

Why use pages = []
I would first start with comprehension list, then add the domain url.